src/core/Table/Table.tsx

```diff
@@ -333,6 +392,8 @@ export const Table = <
                 intersectionMargin={intersectionMargin}
                 state={state}
                 key={rowProps.key}
+                expandedSubComponent={expandedSubComponent}
+                isExpanded={row.isExpanded}
```

Comment:
Sure need separate prop `isExpanded`? you can read the value from the row itself inside `TableRowMemoized`.

Reply:
I previously tried to just use row and would prefer that option, however React.memo prop check for some reason was skipping updates since `prevProp.row.isExpanded` was always equal to `nextProp.row.isExpanded` . I added `isExpanded` separately to fix this issue and now it seems `nextProp.isExpanded` and `prevProp.isExpanded` are actually different.

I'd like more insight on why it is behaving like that, so for now I passed additional prop like that...

Reply:
@bentleyvk thats how it should work?

Reply:
We had the same issue with select, you can check like this but with `expanded`:
![image](https://user-images.githubusercontent.com/36186912/118260232-b34cbe80-b4ba-11eb-925d-6981839e49ba.png)

Reply:
Changed to what you suggested and it does work!

Reply:
I guess it is not needed anymore.

Reply:
Removed it